Tasks/GoToolV0/task.json

```diff
@@ -50,6 +50,14 @@
             "label": "GOBIN",
             "helpMarkDown": "A custom value for the GOBIN environment variable.",
             "groupName": "advanced"
+        },
+        {
+            "name": "goSource",
```

Comment:
Hey @shueybubbles ; instead of free input , this can be a dropdown to choose between two URLs. (as you're not allowing others in code itself.

Reply:
@v-mohithgc will help for this update.

Reply:
thx for the suggestion! Is there an example task that uses a dropdown I should look at?